`make release` fails

## Current Behavior
`make release` fails with
```
added 1512 packages from 686 contributors and audited 1513 packages in 36.512s

253 packages are looking for funding
  run `npm fund` for details

found 2 vulnerabilities (1 moderate, 1 high)
  run `npm audit fix` to fix them, or `npm audit` for details

> neofs_status_page@1.0.0 build /usr/src/app
> BUILD_PATH='./status.fs.neo.org' GENERATE_SOURCEMAP=false react-scripts build

Creating an optimized production build...
One of your dependencies, babel-preset-react-app, is importing the
"@babel/plugin-proposal-private-property-in-object" package without
declaring it in its dependencies. This is currently working because
"@babel/plugin-proposal-private-property-in-object" is already in your
node_modules folder for unrelated reasons, but it may break at any time.

babel-preset-react-app is part of the create-react-app project, which
is not maintianed anymore. It is thus unlikely that this bug will
ever be fixed. Add "@babel/plugin-proposal-private-property-in-object" to
your devDependencies to work around this error. This will make this message
go away.
  
Failed to compile.

[eslint] Failed to load parser '@typescript-eslint/parser' declared in 'package.json » eslint-config-react-app#overrides[0]': Cannot find module 'typescript'
Require stack:
- /usr/src/app/node_modules/@typescript-eslint/typescript-estree/dist/convert.js
- /usr/src/app/node_modules/@typescript-eslint/typescript-estree/dist/ast-converter.js
- /usr/src/app/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /usr/src/app/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /usr/src/app/node_modules/@typescript-eslint/parser/dist/parser.js
- /usr/src/app/node_modules/@typescript-eslint/parser/dist/index.js
- /usr/src/app/node_modules/@eslint/eslintrc/dist/eslintrc.cjs


npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! neofs_status_page@1.0.0 build: `BUILD_PATH='./status.fs.neo.org' GENERATE_SOURCEMAP=false react-scripts build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the neofs_status_page@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2024-02-26T20_31_33_480Z-debug.log
make: *** [Makefile:14: status.fs.neo.org] Ошибка 1
```

## Expected Behavior
Successfully built tarball.

## Possible Solution
Fix it.

## Steps to Reproduce
```
$ make clean
$ make release
```

## Context
Online version is not in sync with repo because of that.

## Regression
Probably related to #98.

## Your Environment
* Version of the product used: master

--- a/package.json
+++ b/package.json
@@ -29,6 +29,7 @@
     "not op_mini all"
   ],
   "devDependencies": {
-    "react-scripts": "^5.0.1"
+    "react-scripts": "^5.0.1",
+    "typescript": "^3.2.1"
   }
 }

